constructor/header.sh

```diff
@@ -5,10 +5,12 @@
 # PLAT:  __PLAT__
 # MD5:   __MD5__
 
+set -eu
```

Comment:
The other scripts have `-o pipefail` too. Would that work here, or do we depend on that behavior?

Reply:
what can be set depends on the shebang... `#!/bin/sh` in this case does not support `-o pipefail` (a very nice feature that requires bash). shellcheck also tells if shebang and set options are incompatible.

Reply:
TIL! Nice, nice. Thanks for the details.